generated/nidcpower/session.py

```diff
@@ -3407,7 +3409,7 @@ def _set_attribute_vi_string(self, attribute_id, attribute_value):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
-    def set_sequence(self, values, source_delays, size):
+    def set_sequence(self, source_delays, size, values=''):
```

Comment:
values should be None or [], not blank string. values is an array!

Reply:
Fixed